### PR TITLE
Implant Breaker Changes

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
They now just break implants and deconvert gangsters. You will have to use your pen on them afterwards to convert. 

They also cost 15 points now

Hopefully this will stop gang mode being a rush to convert security and stop gangs spiralling massively out of control by bypassing the pen cooldown.

If this makes implant spam by security too effective again I'll undo this/try something else.

### CODE REVIEW

**Comment by ExcessiveUseOfCobblestone** [comment]
:+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1:

**Comment by Durkel** [comment]
I think 20 for a implant breaker would be a better price, but we will see how it goes.

**Comment by Supermichael777** [comment]
finally.
honestly non gang leaders being able to use them should have been considered a bug.

**Comment by Boredone** [comment]
Minor complaint, this change makes it so it doesn't deconvert enemy gangers (I think), which was another use.

**Comment by KorPhaeron** [comment]
@Boredone 

I think it's probably better that gangs can't easily consume other gangs, but I could make it deconvert gangsters.

**Comment by PKPenguin321** [comment]
i think it would be good to trial having it deconvert other gangsters, keeps people in the round

**Comment by Lumipharon** [comment]
>i think it would be good to try having it deconvert other gangsters, keeps people in the round

**Comment by Jalleo** [comment]
Yeah let it deconvert other gangs otherwise we will end up with gangs getting loyalty from cargo buying implant breakers then using pens not intuitive.

**Comment by Zilenan91** [comment]
I think 30 is a bit too much for it. If it was 20 or 25 it would be really worth it.

**Comment by Cookiesndip** [comment]
30 points? Holy shit, that's half as expensive as a fucking uzi. Try 15 or 20, don't shoot for the moon.

Actually, I'd suggest doing this incrementally - keep it at 10 TC and change it to deconvert. Suddenly you're limited by your pens, but at the same time you don't have to exhaust ALL your influence for fucking implants that are basically useless. Sec will also roll the gangs.

Remember, nerfing them too hard is going to ruin the mode. I know I'd rather kill and space people at a 30 influence cost for the gear, rather than convert them, especially since at the current cost competing with just the loyalty implants is tough. It's going to turn the mode into a bloodbath, with sec having a huge advantage compared to the gang v.s. gang combat we have now. Keep your hands off Ikkarus' pie, I trust him to make the right call, since he's the one who initially made and balanced the mode

**Comment by Supermichael777** [comment]
30 would be the right price if they hadn't been changed at all. now its just usurers

**Comment by Robustin** [comment]
20 seems fair, thats double the price from before and you can't spam @ 20.

30 seems a little too crippling.

File: code/game/gamemodes/gang/recaller.dm [hunk 1]
```diff
@@ -143,7 +143,7 @@
 			dat += "C4 Explosive<br>"
 
 		dat += "(10 Influence) "
```
Comment (GrayRachnid) [inline code/game/gamemodes/gang/recaller.dm]: Shouldn't this be changed to 30 as well

**Comment by Shadowlight213** [comment]
30 is too expensive for these changes

**Comment by Lumipharon** [comment]
Please merge.

**Comment by Ikarrus** [comment]
My opinion really shouldn't matter anymore, but for what it's worth I'm cool with this

File: code/game/gamemodes/gang/gang_pen.dm [hunk 2]
```diff
@@ -91,24 +91,9 @@
...
-				success = 1	//Not a gangster, convert as usual
-
-		if(!target.mind)
-			return 0
-
-		if(ishuman(target))
-			if(success && ticker.mode.add_gangster(target.mind,gang,0))
-				target.Paralyse(5)
-			else
-				target.visible_message("<span class='warning'>[target] seems to resist the implant!</span>", "<span class='warning'>You feel the influence of your enemies try to invade your mind!</span>")
```
Comment (Ikarrus) [inline code/game/gamemodes/gang/gang_pen.dm]: I don't think you should be getting rid of this, though

File: code/game/gamemodes/gang/gang_pen.dm [hunk 3]
```diff
@@ -91,24 +91,9 @@
...
-				if(ticker.mode.remove_gangster(target.mind,0,1))
-					success = 1	//Was not a gang boss, convert as usual
-			else
-				success = 1	//Not a gangster, convert as usual
-
-		if(!target.mind)
-			return 0
-
-		if(ishuman(target))
-			if(success && ticker.mode.add_gangster(target.mind,gang,0))
```
Comment (Ikarrus) [inline code/game/gamemodes/gang/gang_pen.dm]: add_gangster() was really all you needed to remove

**Comment by KorPhaeron** [comment]
I'll redo it in a bit/lower the cost then

**Comment by Lumipharon** [comment]
Can this please get finshed/merged?

Literally every gang round is just a rush to convert sec first.

**Comment by KorPhaeron** [comment]
I'll finish it tonight in a couple hours

**Comment by Miauw62** [comment]
its not finished?

**Comment by KorPhaeron** [comment]
It is now @Miauw62

**Comment by KorPhaeron** [comment]
Maybe I should do a changelog though

**Comment by Miauw62** [comment]
add changelog then so i can merge this

**Comment by KorPhaeron** [comment]
@Miauw62 

Alright

**Pull request merged**